website/docs/docs/cloud-integrations/set-up-snowflake-native-app.md

```diff
@@ -37,7 +37,7 @@ The following are the prerequisites for dbt Cloud and Snowflake.
 
 - You have **ACCOUNTADMIN** access in Snowflake.
 - Your Snowflake account must have access to the Native App/SPCS integration (PrPr until Summit) and NA/SPCS configurations (PuPr at end of June). If you're unsure, please check with your Snowflake account manager.
```

Comment:
thanks @amychen1776 ! will get this merged for you ! quick question - what does PrPr and PuPr mean here: "PrPr until Summit" and "PuPr at end of June"?